spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -7,22 +7,19 @@
   include ActionView::Helpers::DateHelper
 
   let(:max_attempts) { IdentityConfig.store.doc_auth_max_attempts }
+  let(:fake_analytics) { FakeAnalytics.new }
+
   before(:each) do
-    allow_any_instance_of(ApplicationController).to receive(:analytics).and_return(@fake_analytics)
+    allow_any_instance_of(ApplicationController).to receive(:analytics).and_return(fake_analytics)
```

Comment:
There's a helper method to stub out analytics:
```suggestion
    stub_analytics
```